js/civs.js

```diff
@@ -1,301 +1,396 @@
-                [PARTHIAN_TACTICS, SHIPWRIGHT, REDEMPTION, HERESY, FERVOR, BLAST_FURNACE,
-                    MASONRY, ARCHITECTURE, GOLD_SHAFT_MINING]);
-            unique([RATTAN_ARCHER, ELITE_RATTAN_ARCHER, CHATRAS, PAPER_MONEY], MONK_PREFIX_ASIAN);
-            break;
-        case "Vikings":
-            enable([], [LONGBOAT, ELITE_LONGBOAT], []);
-            disable([KEEP, BOMBARD_TOWER],
-                [HAND_CANNONEER, HEAVY_CAV_ARCHER, HALBERDIER, HUSSAR, CAMEL_RIDER, HEAVY_CAMEL_RIDER, PALADIN,
-                    FIRE_GALLEY, FIRE_SHIP, FAST_FIRE_SHIP, SIEGE_ONAGER, BOMBARD_CANNON],
-                [PARTHIAN_TACTICS, BLOODLINES, HUSBANDRY, SHIPWRIGHT, KEEP, BOMBARD_TOWER, REDEMPTION,
-                    HERBAL_MEDICINE, SANCTITY, ILLUMINATION, THEOCRACY, PLATE_BARDING_ARMOR,
-                    STONE_SHAFT_MINING, GUILDS]);
-            unique([BERSERK, ELITE_BERSERK, CHIEFTAINS, BERSERKERGANG]);
-            break;
-    }
+
+    let selectedCiv = civsConfig[name];
+
+    let enabled = selectedCiv.enabled;
+    let disabled = selectedCiv.disabled;
+    if (selectedCiv.disableHorses) disableHorses(tree);
+
+    enable(enabled.buildings || [], enabled.units || [], enabled.techs || []);
+    disable(disabled.buildings || [], disabled.units || [], disabled.techs || []);
+    unique(enabled.unique || [], selectedCiv.monkPrefix);
 }
+
+const civsConfig = {
+    Aztecs: {
+        disableHorses: true,
+        enabled: {
+            units: [EAGLE_SCOUT, EAGLE_WARRIOR, ELITE_EAGLE_WARRIOR],
+            unique: [JAGUAR_WARRIOR, ELITE_JAGUAR_WARRIOR, ATLATL, GARLAND_WARS]
+        },
+        disabled: {
+            buildings: [KEEP, BOMBARD_TOWER],
+            units: [HAND_CANNONEER, HALBERDIER, CANNON_GALLEON, ELITE_CANNON_GALLEON, HEAVY_DEMO_SHIP, GALLEON, HEAVY_SCORPION, BOMBARD_CANNON],
+            techs: [THUMB_RING, PARTHIAN_TACTICS, HOARDINGS, RING_ARCHER_ARMOR, MASONRY, ARCHITECTURE, BOMBARD_TOWER, KEEP, TWO_MAN_SAW, GUILDS]
+        },
+        monkPrefix: MONK_PREFIX_MESO
+    },
+    Berbers: {
+        enabled: {
+            units: [GENITOUR, ELITE_GENITOUR],
+            unique: [CAMEL_ARCHER, ELITE_CAMEL_ARCHER, KASBAH, MAGHRABI_CAMELS]
+        },
+        disabled: {
+            buildings: [BOMBARD_TOWER, KEEP],
+            units: [ARBALESTER, HALBERDIER, PALADIN, SIEGE_RAM, SIEGE_ONAGER],
+            techs: [PARTHIAN_TACTICS, SHIPWRIGHT, SANCTITY, BLOCK_PRINTING, SAPPERS, ARCHITECTURE, BOMBARD_TOWER, KEEP, TWO_MAN_SAW]
+        },
+        monkPrefix: MONK_PREFIX_AFRICAN
+    },
+    Britons: {
+        disabled: {
+            buildings: [BOMBARD_TOWER],
+            units: [HAND_CANNONEER, HUSSAR, PALADIN, CAMEL_RIDER, HEAVY_CAMEL_RIDER, ELITE_CANNON_GALLEON, SIEGE_RAM, SIEGE_ONAGER, BOMBARD_CANNON],
+            techs: [THUMB_RING, PARTHIAN_TACTICS, BLOODLINES, REDEMPTION, ATONEMENT, HERESY, BOMBARD_TOWER, TREADMILL_CRANE, STONE_SHAFT_MINING, CROP_ROTATION]
+        },
+        enabled: {
```

Comment:
Can we get a consistent ordering of these keys? Sometimes `enabled` comes first, sometimes it's `disabled` like here.